app/models/application_letter.rb

```diff
@@ -20,7 +20,23 @@ class ApplicationLetter < ActiveRecord::Base
   validates :grade, presence: true, numericality: { only_integer: true }
   validates :vegeterian, :vegan, :allergic, inclusion: { in: [true, false] }
   validates :vegeterian, :vegan, :allergic, exclusion: { in: [nil] }
+  validate :deadline_cannot_be_in_the_past, :if => Proc.new { |letter| !(letter.status_changed?) }
```

Comment:
I understand, thank you! I suppose a `lambda` would have been more "clear" for an inexperienced Rails developer like me. But anyways, good work!